ai/core/UnityCatalog_Functions_Quickstart.ipynb

```diff
@@ -0,0 +1,737 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "453a8807-2e81-4164-b310-fd65028f4f1a",
+   "metadata": {},
+   "source": [
+    "# GenAI Usage of UC functions as Agent Tools\n",
+    "\n",
+    "In this tutorial, we will be walking through the process of how to get started with using [Unity Catalog](https://www.unitycatalog.io/)'s function storage capabilities to serve as a GenAI Agent tool repository. \n",
+    "\n",
+    "**Prerequisites for this tutorial**:\n",
+    "\n",
+    "1. A clone of the [Unity Catalog repository](https://github.com/unitycatalog/unitycatalog).\n",
+    "2. JDK-17 installed on your system (in order to build and run the Unity Catalog services)<sup>1</sup>\n",
+    "3. A Python installation version >= 3.9\n",
+    "4. [Docker Desktop](https://www.docker.com/products/docker-desktop/) (recommended; you can also install the docker engine yourself)\n",
+    "\n",
+    "<sup>1</sup> (New to managing Java environments? [jenv](https://github.com/jenv/jenv) is a fantastic tool that can help!) \n",
+    "\n",
+    "**Package Requirements (Python)**:\n",
+    "\n",
+    "You can install the required packages for this tutorial by running:\n",
+    "\n",
+    "```shell\n",
+    "pip install unitycatalog-client unitycatalog-ai unitycatalog-openai\n",
+    "```\n",
+    "\n",
+    "## Objectives\n",
+    "\n",
+    "By the end of this tutorial you will have learned:\n",
+    "\n",
+    "- How to start a Unity Catalog server using Docker\n",
+    "- How to use the `unitycatalog-client` to create a Catalog and a Schema to store your functions\n",
+    "- How to create Unity Catalog Functions to store Python callables\n",
+    "- How to list functions available in a given Catalog and Schema using the `unitycatalog-client` package\n",
+    "- How to execute stored functions for validation\n",
+    "- How to create a UCFunctionToolkit instance to register functions as tools for use in calls to OpenAI\n",
+    "- How to build a very simple tool calling Agent using OpenAI and Unity Catalog\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bafb35c7-ca83-4c73-96db-ccfdeb094cfa",
+   "metadata": {},
+   "source": [
+    "## Starting a UC server\n",
+    "\n",
+    "After you've cloned the Unity Catalog repository, navigate to the repository root (`unitycatalog`). Once there, simply run:\n",
```

Comment:
Maybe provide a command here `git clont ...`; also the image will be published right? so we can simplify this tutorial later

Reply:
Yep once 0.2.1 is released with support for function calling and the Docker container is hosted, we'll update this quickstart and the docs :)